main_test.go

```diff
@@ -0,0 +1,357 @@
+/*
```

Comment:
License header format: https://github.com/banzaicloud/pipeline/blob/be270fb/helm/helm.go#L1

Reply:
changed to the line comment format